Mathlib/Util/CountHeartbeats.lean

```diff
@@ -0,0 +1,64 @@
+/-
+Copyright (c) 2023 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import Mathlib.Lean.CoreM
+import Mathlib.Data.KVMap
+import Std.Tactic.TryThis
+import Mathlib.Tactic.LibrarySearch
```

Comment:
Can you clean the imports up a bit? I think that `Util` should generally not import things from `Tactic`.